panels/templates/panel_applications/index.html

```diff
@@ -0,0 +1,246 @@
+{% extends "preregistration/preregbase.html" %}
+{% block title %}Panel Applications{% endblock %}
+{% block backlink %}{% endblock %}
+{% block content %}
+
+<script>
+    var showOrHideHeldBefore = function () {
```

Comment:
We have a lot of these kinds of functions, both here and in core - it might be worthwhile to make a utility function that takes the field-to-hide name and the condition instead so as to reduce clutter.